build-settings/rollup.config.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable import/no-commonjs */
+import fs from "fs";
+import autoExternal from "rollup-plugin-auto-external";
+import babel from "@rollup/plugin-babel";
+import {terser} from "rollup-plugin-terser";
+
+const {presets, plugins} = require("./babel.config.js");
+
+const createConfig = (pkgName) => ({
+    output: {
+        file: `packages/${pkgName}/dist/index.js`,
+        format: "esm",
+    },
+    input: `packages/${pkgName}/src/index.js`,
+    plugins: [
+        babel({
+            babelHelpers: "bundled",
+            presets,
+            plugins,
+            exclude: "node_modules/**",
```

Comment:
I assume this also works for imports of other `ws-*` modules from one another since we aren't doing any module mapping shenanigans here.

Reply:
I think so. This is based on WB config